README.md

```diff
@@ -1,122 +1,145 @@
+With Docker, the files within the container are not persisted. To persist data, you will want to mount a directory from the host to the container's /opt/aerospike/data using the `-v` option:
 
-To specify **access-address** in aerospike.conf:
+For example:
 
-	network {
-		service {
-			address any                  # Listening IP Address
-			port 3000                    # Listening Port
-			access-address 192.168.1.100 # IP Address to be used by applications
-																	 # and other nodes in the cluster.
-		}
-		...
-
-
-## Persistent Data Directory
+```sh
+docker run -tid  -v /opt/aerospike/data:/opt/aerospike/data  -v /opt/aerospike/etc:/opt/aerospike/etc/ --name aerospike -p 3000:3000 -p 3001:3001 -p 3002:3002 -e "FEATURE_KEY_FILE=/opt/aerospike/etc/features.conf" aerospike/aerospike-server-enterprise
+```
 
-With Docker, the files within the container are not persisted. To persist the data, you will want to mount a directory from the host to the guest's `/opt/aerospike/data` using the `-v` option:
+The example above uses the configuration template, where the single defined
+namespace is in-memory with file-based persistence. Just mounting the predefined
+/opt/aerospike/data directory enables the data to be persisted on the host.
 
-	-v <DIRECTORY>:/opt/aerospike/data
+In an alternate case a custom configuration file is used with the parameter
+`file` set to be a file in the mounted /opt/aerospike/data, such as in this config snippet
 
-Where `<DIRECTORY>` is the path to a directory containing your data files.
+```
+namespace test {
+	# :
+	storage-engine device {
+		file /opt/aerospike/data/test.dat
+		filesize 4G
+		data-in-memory true
+	}
+}
+```
 
-A full example:
+In this example we also mount the data directory in a similar way, using a
+custom configuration file
 
-	docker run -tid -v <DIRECTORY>:/opt/aerospike/data --name aerospike -p 3000:3000 -p 3001:3001 -p 3002:3002 -p 3003:3003 -v <DIRECTORY>:/etc/aerospike/ -e "FEATURE_KEY_FILE=/etc/aerospike/features.conf" aerospike/aerospike-server-enterprise
+```sh
+docker run -tid -v /opt/aerospike/data:/opt/aerospike/data -v /opt/aerospike/etc/:/opt/aerospike/etc/ --name aerospike -p 3000:3000 -p 3001:3001 -p 3002:3002 aerospike/aerospike-server-enterprise --config-file /opt/aerospike/etc/aerospike.conf
+```
 
-## Block Storage
+### Block Storage
 
-Docker run offers the ability to expose hosts block devices to a running container. The --device option can be used to map a host block device within a container.
+Docker run offers the ability to expose hosts block devices to a running container. The `--device` option can be used to map a host block device within a container.
 
 Example mapping /dev/sdc to /dev/xvdc on a running container.
 
+```sh
+docker run -tid --name aerospike -p 3000:3000 -p 3001:3001 -p 3002:3002 -v /aerospike-server-enterprise.docker:/etc/aerospike/ --device '/dev/sdc:/dev/xvdc' -e "FEATURE_KEY_FILE=/etc/aerospike/features.conf" aerospike-server-enterprise:latest
```

Comment:
@volmarl do you know what this /aerospike-server-enterprise.docker path in the -v mapping is about? It's confusing to me, but I wasn't sure what to replace it with

Reply:
It could be any local folder <Local Absolute path> where docker run is running.